lib/immutable-struct.rb

```diff
@@ -81,8 +81,25 @@ def self.new(*attributes,&block)
         end
       end
 
+      @@aliases = {}
+
+      def self.alias_attribute(aliaz, attr, &block)
+        @@aliases.store(aliaz, [attr, block])
+      end
+
+      def method_missing(m, *_args, &_block)
```

Comment:
The build failed initially with the signature `method_missing(m, *_, &_)`. I assume it's because Ruby doesn't like the underscores in this case, but I didn't dig all the way down on the cause of failure.